firefox-ios/firefox-ios-tests/Tests/XCUITests/DownloadsTests.swift

```diff
@@ -118,9 +118,13 @@ class DownloadsTests: BaseTestCase {
         shareButton.tap(force: true)
         mozWaitForElementToExist(app.tables["DownloadsTable"])
         mozWaitForElementToExist(app.tables["DownloadsTable"].staticTexts[testFileNameDownloadPanel])
-        mozWaitForElementToExist(app.collectionViews.cells["Copy"])
+        if #available(iOS 16, *) {
+            mozWaitForElementToExist(app.collectionViews.cells["Copy"])
+        } else {
+            mozWaitForElementToExist(app.collectionViews.buttons["Copy"])
+        }
         if !iPad() {
-            app.otherElements.element(boundBy: 1).buttons["Close"].tap()
+            app.navigationBars["UIActivityContentView"].buttons["Close"].tap()
```

Comment:
`app.navigationBars["UIActivityContentView"].buttons["Close"]` works on iOS 15, 16 and 17.